lib/ansible/modules/cloud/azure/azure_rm_servicebus_facts.py

```diff
@@ -95,227 +95,269 @@
 '''
 RETURN = '''
 servicebuses:
-    description: List of servicebus dicts.
+    description:
+        - List of servicebus dicts.
     returned: always
-    type: list
+    type: complex
```

Comment:
list

Reply:
if describe nested return values, set type: complex and repeat the elements above for each sub-field (Azure format [document's link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html)).